apps/browser/components/Sidebar.tsx

```diff
@@ -260,7 +272,15 @@ const Sidebar = ({
                   )}
                 </div>
               )}
-              <CurrentWorkspace />
+              <Workspace
+                onUpload={(uploads: InMemoryFile[]) => {
```

Comment:
Can we move to logic to the top of the component instead of having it with the hmtl?

Reply:
Sure thing, just did this to reduce complexity of the component's function body while I was refactoring.